frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -46,6 +59,12 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
         <div className="table-item">{returnFastestShot(TeamID.Team_blue)}</div>
         <div className="table-item">fastest shot of the game</div>
         <div className="table-item">{returnFastestShot(TeamID.Team_white)}</div>
+        <div className="table-item">{getManualAddedGoals(TeamID.Team_blue)}</div>
+        <div className="table-item">Manually added goals</div>
+        <div className="table-item">{getManualAddedGoals(TeamID.Team_white)}</div>
+        <div className="table-item">{getManualSubstractedGoals(TeamID.Team_blue)}</div>
+        <div className="table-item">Manually substracted goals</div>
+        <div className="table-item">{getManualSubstractedGoals(TeamID.Team_white)}</div>
```

Comment:
Consider (now or later) to create some abstraction for these divs. You will have more and more statistics, so the list of divs will grow as well. That won't look good.

Reply:
For now, I leave it as is. It will be fixed in separate PR